README.md

```diff
@@ -78,3 +79,5 @@ Read more about [JSON to JSON transformations](https://www.fluvio.io/smartmodule
 
 Follow on the conventional `CONTRIBUTING.md` file to setup your environment and
 contribute to this project.
+
+[1]: https://www.fluvio.io/connectors/outbound/http/
```

Comment:
This page progress is tracked here: https://github.com/infinyon/fluvio-website/issues/560